extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -1864,6 +1875,119 @@ describes.realWin('Events', {amp: 1}, env => {
       });
     });
 
+    describe('multi selector visibility trigger', () => {
```

Comment:
can this describe set be combined with related unit tests around selector and target. You can move existing tests to this block for better grouping.